### PR TITLE
Add blog post for StarlingX PTP redundant timing clock sources

### PR DESCRIPTION
**Pull request by andrezelak-wind**
Adds markdown files for the blog post and author information.

### CODE REVIEW

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+title: Redundant PTP timing clock sources
+author: Andre Mauricio Zelak
+date: 2024-03-25
+category:
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+In the rapidly evolving telecommunications landscape, 5G stands as a beacon of innovation, promising unparalleled connectivity and transformative capabilities. With its potential to revolutionize industries ranging from healthcare to manufacturing, the rollout of 5G networks has accumulated immense attention and anticipation. Beneath the surface of this technology lies an indispensable element, the Precision Time Protocol (PTP).
+
+And as part of the constant effort to meet the evolving needs of telecommunications industry and to provide a more robust Precision Time Protocol (PTP) deployment, the StarlingX 9.0 introduces the High Availability feature, providing a way to sync multiple Physical Hardware Clock (PHC) in a N+1 redundancy configuration.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:12]:
```suggestion
And as part of the constant effort to meet the evolving needs of telecommunications industry and to provide a more robust Precision Time Protocol (PTP) deployment, StarlingX 9.0 introduces the High Availability (HA) feature, providing a way to sync multiple Physical Hardware Clock (PHC) in a N+1 redundancy configuration.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+The feature allows, in a t-GM application, multiple GPS clock sources connected, and in a t-BC application, multiple PTP sources connected to redundant NICs, provinding protection against failtures.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:14]:
```suggestion
The feature allows, in a t-GM application, multiple GPS clock sources connected, and in a t-BC application, multiple PTP sources connected to redundant NICs, providing protection against failures.
```

Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:14]: Can you please also briefly describe t-GM and t-BC applications? Or add a pointer to a good resource that does? That will make the blog post more accessible to a wider range of audience.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+This article will speak about the new HA feature configuration, clock selection algorithm, and new alarms.
+
+If you're interested in learning more about the initial use cases and implementation, you check out an earlier [blog post series](https://www.starlingx.io/blog/starlingx-ptp-part-1/) and ["A Deep Dive Into the Precision Time Protocol Configuration Enhancements"](https://www.starlingx.io/blog/starlingx-ptp-multi-instance-features/).
+
+# Configuration
+
+As pointed out in ["A Deep Dive Into the Precision Time Protocol Configuration Enhancements"](https://www.starlingx.io/blog/starlingx-ptp-multi-instance-features/), when a StarlingX host is configured as a PTP node,  there are two applications used: ptp4l and phc2sys. The ptp4l is responsible for running the Best Master Clock Algorithm (BMCA) and to desciplin the oscilator on the NIC according to the incomming PTP timestamps. And the phc2sys reads the time from a disciplined NIC and synchronizes the system clock with it.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:24]:
```suggestion
As pointed out in the ["A Deep Dive Into the Precision Time Protocol Configuration Enhancements"](https://www.starlingx.io/blog/starlingx-ptp-multi-instance-features/) article, when a StarlingX host is configured as a PTP node,  there are two applications used: ptp4l and phc2sys. ptp4l is responsible for running the Best Master Clock Algorithm (BMCA) and to discipline the oscillator on the NIC according to the incoming PTP timestamps. And phc2sys reads the time from a disciplined NIC and synchronizes the system clock with it.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+Up to StarlingX version 8.0, the Linux real-time system clock was synchronized to only a single time source (PHC) through static configuration of the phc2sys instance. From release 9.0 and up, when the HA feature enabled, the phc2sys configuration will accept association to multiple ptp4l instances, providing redundancy protection to the clock sources.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:26]:
```suggestion
Up to StarlingX version 8.0, the Linux real-time system clock was synchronized to only a single time source (PHC) through static configuration of the phc2sys instance. From release 9.0 and up, when the HA feature is enabled, the phc2sys configuration will accept association to multiple ptp4l instances, providing redundancy protection to the clock sources.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+When the HA feature enabled, the real-time system clock is synchronized with the best time source available. The operator configure the following clock status values to control the source selection algorithm:
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:28]:
```suggestion
When the HA feature is enabled, the real-time system clock is synchronized with the best time source available. The operator configures the following clock status values to control the source selection algorithm:
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+* GM clock class
+* GM clock accuracy
+* GM offset scaled log variance
+* Local clock class
+* Local clock accuracy
+* Local offset scaled log variance
+* Frequency traceable
+* Time traceable
+
+These parameters set the threshold for a source to be eligible for use in setting the system real-time clock. For example, in a t-GM application, configuring 6 in the maximum local class makes only locked sources eligible for selection.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:38]: Can you please explain this in a bit more detail?

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+The operator can also configure a clock priority to indicate a precedence, which is usefull when the sources are not equal in reliability. The clock priority is a tiebreaker for sources of equal quality. The higher priority clock is selected active over the lower one when both meet the requirements.
+
+In addition, the HA feature enables the phc2sys to accept multiple PTP domains. Every ptp4l can be associate to a different PTP domain. See 'ha_domainNumber' interface parameter in the documentation.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:42]:
```suggestion
In addition, the HA feature enables the phc2sys to accept multiple PTP domains. Every ptp4l can be associated to a different PTP domain. See 'ha_domainNumber' interface parameter in the documentation.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+The ["StarlingX Documentation"](https://docs.starlingx.io/system_configuration/kubernetes/configuring-ptp-service-using-the-cli.html#ptp-instance-configuration) contains details about the new configuration fields, their default values and ranges.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:44]:
```suggestion
The [StarlingX Documentation](https://docs.starlingx.io/system_configuration/kubernetes/configuring-ptp-service-using-the-cli.html#ptp-instance-configuration) contains details about the new configuration fields, their default values and ranges.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+# Clock selection
+
+At start, the best available clock is selected active. When the active clock degrades and doesn't meet the requirements,  another eligible clock is selected active. And when a clock recovers and meets the requirement, it becomes elible again.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:48]:
```suggestion
At start, the best available clock is selected active. When the active clock degrades and doesn't meet the requirements,  another eligible clock is selected active. And when a clock recovers and meets the requirements, it becomes eligible again.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+Depending on the clock priorities, the clock switch behavior is different. There are two broad scenarios: equal or different priorities.
+
+In a scenario where all clocks are equal priority, when the active clock deteriorate and another clock satisfies the requirements, this other clock is set active. Than, when the degraded clock recovers, it becomes eligible, but in stand by.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:52]:
```suggestion
In a scenario where all clocks are equal priority, when the active clock deteriorates and another clock satisfies the requirements, this other clock is set active. Then, when the degraded clock recovers, it becomes eligible, but in stand by.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+Now, in case of different priorities. The higher priority clock has precedence over the lower one. If both are good, the higher priority one is selected active. After an outage, when the higher priority clock heal, it becomes active.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:54]:
```suggestion
Now, in case of different priorities. The higher priority clock has precedence over the lower one. If both are good, the higher priority one is selected active. After an outage, when the higher priority clock heals, it becomes active.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+
+The administrator can configure a stability timer to prevent multiple and consecutive clock switches, when the higher priority clock fail and come back. When the selected clock degrade, another clock takes place quickly. But, when the higher priority recovers, it will only become active after the stability time expiration.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:56]:
```suggestion
The administrator can configure a stability timer to prevent multiple and consecutive clock switches, when the higher priority clock fails and comes back. When the selected clock degrades, another clock takes place quickly. But, when the higher priority recovers, it will only become active after the stability time expiration.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+
+During an outage event, when all the sources fail to satisfy the requirements, one of the sources is set active as fallback. The clock with the lowest clock class is selected. As a tiebreaker, if more than one source present the same lowest local clock class, the one configured first is selected.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:58]:
```suggestion
During the event of an outage, when all the sources fail to satisfy the requirements, one of the sources is set active as fallback. The clock with the lowest clock class is selected. As a tiebreaker, if more than one source present the same lowest local clock class, the one configured first is selected.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+
+# Applications
+
+The HA feature is totally transparent for applications using the system real-time clock source. Neither the clock selection nor the clock switching should be a concern. And no application change is required.
+
+# ptp-notification registration and reporting application
+
+The ptp-notification registration and reporting applications support multiple clock sources in compliance to O-RAN O-Cloud Notification API Specification. An aggregated clock status is also supported. The user application may need enhancements to accomodate a list of clock sources.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:66]:
```suggestion
The ptp-notification registration and reporting applications support multiple clock sources in compliance to the O-RAN O-Cloud Notification API Specification. An aggregated clock status is also supported. The user application may need enhancements to accommodate a list of clock sources.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+
+# New alarms
+
+New alarms and events have been added to StarlingX to report the HA state.
+
+The "no source clock" alarm raises when phc2sys does not have sources that meet the specified threshold in the user configuration.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:72]:
```suggestion
The "no source clock" alarm is raised when phc2sys does not have sources that meet the specified threshold in the user configuration.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+
+The "source clock is not locked to a PRC" alarm raises when one of the HA phc2sys interfaces presents a clockClass of lesser quality than the specfied maximum clockClass in the user config. The alarm indicates the degraded interface.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:74]:
```suggestion
The "source clock is not locked to a PRC" alarm is raised when one of the HA phc2sys interfaces presents a clockClass of lesser quality than the specified maximum clockClass in the user config. The alarm indicates the degraded interface.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+
+The "selected lower priority clock source" alarm raises when phc2sys is configured with multiple sources, and the active is not the highest priority. It indicates the highest priority clock has degraded.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:76]:
```suggestion
The "selected lower priority clock source" alarm is raised when phc2sys is configured with multiple sources, and the active is not the highest priority. It indicates the highest priority clock has degraded.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+
+The "automatic source selection disabled" alarm raises when an operator locks the system real-time clock to an interface. See "status & debuging" below.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:78]:
```suggestion
The "automatic source selection disabled" alarm is raised when an operator locks the system real-time clock to an interface. See "status & debugging" below.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+
+The "selected new active source" event raises when phc2sys switches between sources for any reason.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:80]:
```suggestion
The "selected new active source" event is raised when phc2sys switches between sources for any reason.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 20]
```diff
@@ -0,0 +1,92 @@
...
+
+The ["StarlingX Documentation"](https://docs.starlingx.io/system_configuration/kubernetes/configuring-ptp-service-using-the-cli.html#ptp-instance-configuration) contains additional information about these alarms and also how to investigate them.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:82]:
```suggestion
The [PTP instance configuration](https://docs.starlingx.io/system_configuration/kubernetes/configuring-ptp-service-using-the-cli.html#ptp-instance-configuration) section in the StarlingX documentation contains additional information about these alarms and how to investigate them.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 21]
```diff
@@ -0,0 +1,92 @@
...
+
+# status & debuging
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:84]:
```suggestion
# Status & debugging
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 22]
```diff
@@ -0,0 +1,92 @@
...
+
+The HA feature provides an interface for maintenance, deemed for the host admin to retrieve status and control the clock list and selection. The admin can:
+* Display the current clock status, including all the values used by the clock selection algorithm;
+* List the eligible for selection clocks;
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:88]:
```suggestion
* List the eligible clocks for selection;
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 23]
```diff
@@ -0,0 +1,92 @@
...
+* Disable and re-enable a clock source;
+* Lock to a single clock source and unlock.
+
+You can find more information about this feature in the ["StarlingX Documentation"](https://docs.starlingx.io/system_configuration/kubernetes/configuring-ptp-service-using-the-cli.html#ptp-instance-configuration).
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:92]:
```suggestion
You can find more information about this feature in the ["StarlingX Documentation"](https://docs.starlingx.io/system_configuration/kubernetes/configuring-ptp-service-using-the-cli.html#ptp-instance-configuration).

# About StarlingX

If you would like to learn more about the project and get involved check the [website](https://www.starlingx.io) for more information or [download the code](https://opendev.org/starlingx) and start to experiment with the platform. If you are already evaluating or using the software please fill out the [user survey](https://openinfrafoundation.formstack.com/forms/starlingx_user_survey) and help the community improve the project based on your feedback.
```

**Review by ildikov — COMMENTED** [review]
Hi Andre,

Thank you for submitting the article for the StarlingX blog, it looks good overall. I added a few suggestions in line to fix some typos and have some more details for a few items.

I scheduled the blog post for July, to space out the articles that are currently waiting to be published. I will move it up if I get more content in the meantime.

Thanks and Best Regards,
Ildikó

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 24]
```diff
@@ -0,0 +1,92 @@
+---
+templateKey: blog-post
+title: Redundant PTP timing clock sources
+author: Andre Mauricio Zelak
+date: 2024-03-25
```
[the remark this answers is not in the document]
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Acepted the suggested change.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 1 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested change.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 2 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested changes and added a link to a PTP acronyms glossary.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 3 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested change.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 4 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested change.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 19 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested change.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 20 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested change.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 21 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested change.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 22 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested change.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 23 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Accepted the suggested change.

**Comment by andrezelak-wind** [comment]
Hi Ildikó,

I very much appreciate your contribution. I updated the post with all the suggested changes, but I know it might not be all. So, please let me know if you have any additional suggestions.

Thanks and Best Regards,
Andre

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md [hunk 25]
```diff
@@ -0,0 +1,96 @@
...
+title: Redundant PTP timing clock sources
+author: Andre Mauricio Zelak
+date: 2024-07-22
+category:
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+In the rapidly evolving telecommunications landscape, 5G stands as a beacon of innovation, promising unparalleled connectivity and transformative capabilities. With its potential to revolutionize industries ranging from healthcare to manufacturing, the rollout of 5G networks has accumulated immense attention and anticipation. Beneath the surface of this technology lies an indispensable element, the Precision Time Protocol (PTP).
+
+And as part of the constant effort to meet the evolving needs of telecommunications industry and to provide a more robust Precision Time Protocol (PTP) deployment, StarlingX 9.0 introduces the High Availability (HA) feature, providing a way to sync multiple Physical Hardware Clock (PHC) in a N+1 redundancy configuration.
```
Comment (ildikov) [inline src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:12]:
```suggestion
And as part of the constant effort to meet the evolving needs of the telecommunications industry and to provide a more robust PTP deployment, StarlingX 9.0 introduces the High Availability (HA) feature, providing a way to sync multiple Physical Hardware Clock (PHC) in a N+1 redundancy configuration.
```

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 6 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:38]: This sentence is a bit hard to follow. Does the example of configuring the maximum local class refer to 'GM clock class'?

**Review by ildikov — CHANGES_REQUESTED** [review]
Hi Andre,

Thank you for the fixes.

I added a couple more nits in line.

It also appears that a few suggestions in my first review haven't been applied, so there are a some typos in the article still. Did you see those comments?

Thanks and Best Regards,
Ildikó

**Comment by ildikov** [comment]
@andrezelak-wind Hi Andre, I'm reaching out to check if you had a chance to look at the review comments I left on the draft article?

**Comment by andrezelak-wind** [comment]
Hi Ildiko, I was out of office for two weeks, I'm returning Today. I'll take a look at your comments as soon as possible.
________________________________

**Comment by ildikov** [comment]
Hi Andre,

No worries, thank you for looking into it. I will keep an eye out for your updates.

Thanks,
Ildikó

**Comment by andrezelak-wind** [comment]
Hi Ildikó,

The latest upload should include all the suggestions, but if I missed some please let me know.
In specific, I expanded the text about the example I gave. I hope it is clearer now, but I thank any contribution.

Thanks,
 André

**Comment by ildikov** [comment]
Hi @andrezelak-wind,

I just started to review your submission, but it doesn't seem to have the typo fixes that I previously pointed out. Are you sure you uploaded the right version for review?

Thanks,
Ildikó

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 5 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:28]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 7 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:42]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 8 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:44]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 9 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:48]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 10 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:52]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 11 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:54]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 12 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:56]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 13 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:58]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 14 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:66]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 15 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:72]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 16 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:74]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 17 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:76]: This comment is still unresolved.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 18 above)
Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:78]: This comment is still unresolved.

Comment (ildikov) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md:78]: This comment is still unresolved.

**Review by ildikov — CHANGES_REQUESTED** [review]
Hi Andre,

I just looked into the latest build, and in the preview all these comments are still unresolved.

Best Regards,
Ildikó

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Done!

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 1 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Done!

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 2 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Done!

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 3 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Done!

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 4 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: Done!

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 5 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 6 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: No, it refers to 'Local clock class'. It is a bit confuse to use 'Local clock class' in t-GM configuration, not 'GM clock class'. The clock is local for the system configured as t-GM, because it is directly connected to the GPS clock source.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 7 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 8 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 9 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 10 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 11 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 12 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 13 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 14 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 15 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 16 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 17 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 18 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

File: src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md (hunk 19 above)
Comment (andrezelak-wind) [reply src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md]: It was unresolved. Corrected now in the 4th commit.

**Review by ildikov — APPROVED** [review]
Hi Andre,

Thank you for making the updates!

I just updated the 'date' label to schedule the blog post to go live on August 12.

Thank you for contributing the article!

Best Regards,
Ildikó
